Server/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -1637,6 +1650,82 @@ int CLuaElementDefs::removeElementData(lua_State* luaVM)
     return 1;
 }
 
+int CLuaElementDefs::subscribeElementData(lua_State* luaVM)
+{
+    //  bool subscribeElementData ( element theElement, string key, player thePlayer )
+    CElement* pElement;
+    SString   strKey;
+    CPlayer*  pPlayer;
+
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData(pElement);
+    argStream.ReadString(strKey);
+    argStream.ReadUserData(pPlayer);
+
+    if (!argStream.HasErrors())
+    {
+        CLuaMain* pLuaMain = m_pLuaManager->GetVirtualMachine(luaVM);
+        if (pLuaMain)
+        {

```

Comment:
I don't think you need `pLuaMain` checks here

Reply:
But `GetVirtualMachine(luaVM)` can return `NULL` in some cases, isn't it? Moreover functions above also have this check.

Reply:
I actually don't know for sure, but I think that's because CLuaElementDefs has some _really_ old definitions. Plus you're not using `pLuaMain` anywhere anyway